common/entity/ScreeningTypes.ts

```diff
@@ -0,0 +1,19 @@
+export enum JobStatus {
+    STUDENT = 'Schüler:in',
+    PUPIL = 'Student:in',
+    EMPLOYEE = 'Angestellte:r',
+    RETIREE = 'Rentner:in',
+    MISC = 'Sonstige',
+}
+
+export enum KnowsCoronaSchoolFrom {
```

Comment:
This is not in use yet. Any naming suggestions for the database field? Because knowsCoronaSchoolFrom: string already exists

Reply:
I guess we can just continue to use the existing varchar and write some well known strings into it so that COUNT(*) GROUP BY works

Reply:
With writing some well-known strings into it, you mean handling these options in retool without changing anything in the backend right?

Reply:
yes exactly

Reply:
Thats also easier to adapt in the future when requirements change